java/gorpc/src/main/java/com/youtube/vitess/gorpc/codecs/bson/BsonClientCodec.java

```diff
@@ -45,7 +43,7 @@ public void WriteRequest(Request request, Object args) throws IOException {
   public void ReadResponseHeader(Response response) throws IOException {
     BSONObject headerBson = decoder.readObject(socket.getInputStream());
     response.setServiceMethod(new String((byte[]) headerBson.get(Constants.SERVICE_METHOD)));
-    response.setSeq(UnsignedLong.fromLongBits((long) headerBson.get(Constants.SEQ)));
+    response.setSeq((long)headerBson.get(Constants.SEQ));
```

Comment:
This doesn't look right.. setSeq() takes an UnsignedLong, so the conversion should be necessary. Did this work locally for you?

I'm thinking maybe the error you saw in YCSB is actually because the fixed client hasn't been uploaded to our [maven repo](https://github.com/youtube/mvn-repo). Do you know how to do that?